src/FieldWarning/Assets/Model/Game/MatchSession.cs

```diff
@@ -67,8 +67,8 @@ public void Awake()
             var blueTeam = GameObject.Find("Team_Blue").GetComponent<Team>();
             var redTeam = GameObject.Find("Team_Red").GetComponent<Team>();
 
-            blueTeam.AddPlayer(this);
-            redTeam.AddPlayer(this);
+            blueTeam.AddPlayer(this, "player-blue");
+            redTeam.AddPlayer(this, "player-red");
```

Comment:
This is a very hacky way to associate the player to their deck, it really should be passed directly.

Reply:
Yeah, I'm not super happy about this, but I also figured it was kind of a temporary placeholder since eventually we'll be dealing with actual players with actual saved deck configurations instead of just grabbing arbitrary JSON files by player name. But I can definitely look into doing this a little more sanely.